src/firstvoices/backend/models/base.py

```diff
@@ -55,3 +58,12 @@ class Meta:
 
     # from dc:lastContributor
     last_modified = models.DateTimeField(auto_now=True, db_index=True)
+
+
```

Comment:
Does this function need to be called on the parts of speech? I noticed in the admin site I can still edit last created and last modified. If this will be called later to set those default values on parts of speech that's all good.